python/dask_cudf/dask_cudf/tests/test_dataframe.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+
+import cudf
+import dask.dataframe as dd
+import dask_cudf
+
+import pytest
+
+
+def test_get_dummies(data):
```

Comment:
Tests for get_dummies currently live in `tests/test_onehot.py`. For consistency, we should include these there for now.

More broadly, @shwina , given your recent thinking about the directory structure, do you want to draw an organizational line in the in the sand between cudf and dask tied tests in the short/medium term?

Reply:
sure, would move them to `test_onehot.py`. Will wait for @shwina inputs if any other organizational structure is recommended?

Reply:
@galipremsagar can you break this test out into multiple tests instead? That way if something fails we have a much more directed area to investigate.

Reply:
@galipremsagar 

`dask_cudf/tests/test_onehot.py` sounds good to me.

Organizationally, as @beckernick suggests, I do think it makes sense to draw a line between cudf tests and dask tests. Tests for `cudf` live under the `cudf` package and tests for `dask_cudf` under the `dask_cudf` package. We should never have to `import dask_cudf` from anywhere within the `cudf` test suite.

Reply:
@kkraus14 , @shwina refactored accordingly.